webrecorder/browsertrix
# feat: Manage org read-only state + superadmin view refresh

Resolves https://github.com/webrecorder/browsertrix/issues/1903

Note: This was branched from https://github.com/webrecorder/browsertrix/pull/1788 to get ahead of potential conflicts

### Changes

- Displays org status in superadmin view
- Enables superadmins to update org read-only status
- Sorts orgs alphabetically
- Refactors org list to `TailwindComponent`

### Manual testing

1. Log in as superadmin
2. Click "..." action menu for an org
3. Click "Make Read-Only". Verify dialog is shown
4. Enter reason and submit. Verify org is now read-only
5. Click action menu > "Undo Read-Only". Verify org is no longer read-only

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Superadmin Dashboard | <img width="1026" alt="Screenshot 2024-07-01 at 10 13 21 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/0ac304fb-bdc7-405d-af55-9264b3c1fe1f"> |
| Superadmin Dashboard | <img width="601" alt="Screenshot 2024-07-01 at 10 13 28 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/ad80bff1-ca87-4296-9bf8-175df61c3009"> |
| Superadmin Dashboard | <img width="1034" alt="Screenshot 2024-07-01 at 10 13 42 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/4794688a-d408-4500-85c6-5aa974568943">
